### PR TITLE
fix(DB/Quest): Quest "Devourer of Souls" Has Incorrect Qu…

### LINKED ISSUE #6887: Orgrimmar - Quest "Devourer of Souls" Has Incorrect Quest Reward Text
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [ ] Alliance
- [x] Horde

##### CONTENT PHASE:

- [ ] Generic
- [ ] 1-19
- [x] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

Original report: https://github.com/chromiecraft/chromiecraft/issues/1164

Upon talking to Cazul (https://wowgaming.altervista.org/aowow/?npc=5909) for completing Devourer of Souls (https://wowgaming.altervista.org/aowow/?quest=1507), the following quest text appears:

Hm... **Zarm**. You are still new to your path, but I sense the possibility for greatness in you.

You were born with gifts, \<name\>. See that they do not go to waste.

WoWScrnShot_071121_152741

Tester: confirmed. Checking the `quest_offer_reward` table for this quest shows the faulty text. The 'Zarm' needs to be replaced with $N.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

According to https://wowpedia.fandom.com/wiki/Devourer_of_Souls_(Orgrimmar), it should be:

Hm... \<name\>. You are still new to your path, but I sense the possibility for greatness in you.

You were born with gifts, \<name\>. See that they do not go to waste.

##### SOURCE:

https://wowpedia.fandom.com/wiki/Devourer_of_Souls_(Orgrimmar)

##### STEPS TO REPRODUCE THE PROBLEM:

1. Be at least a level 20 warlock.
2. Take the quest Devourer of Souls from Gan'rul Bloodeye (DB GUID: 256).
3. Talk to Cazul (DB GUID: 3419).

Or just look in Keira for quest 1507 -> Completion (RewardText)

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/f16fe3f8dc5e6fd0af8848714fab2195cb1182c1

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/99900436-orgrimmar-quest-devourer-of-souls-has-incorrect-quest-reward-text?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Melonio2**
**Fix DB RewardText for quest 1507**

DESCRIPTION
Change hard coded name to adapt to the player's name.

Closes #6887

DB - (database related, sql)

Changes Proposed:
-  Text modified.

SOURCE:
**Azcobu's work ! He gave the solution in the issue he created so I used to do my first PR. Many thanks to him/her.**

How to Test the Changes: (pasted from Azcobu's issue).

1. Be at least a level 20 warlock.
2.Take the quest Devourer of Souls from Gan'rul Bloodeye (DB GUID: 256).
3.Talk to Cazul (DB GUID: 3419).

### CODE REVIEW

File: data/sql/updates/pending_db_world/create_sql.sh [hunk 1]
```diff
@@ -13,3 +13,7 @@ rev=$( $date +%s%N );
 filename=rev_"$rev".sql
 
 echo "INSERT INTO \`version_db_world\` (\`sql_rev\`) VALUES ('"$rev"');" > "$CUR_PATH/$filename" && echo "File created: $filename";
+
+DELETE FROM `quest_offer_reward` WHERE (`ID` = 1507);
+INSERT INTO `quest_offer_reward` (`ID`, `Emote1`, `Emote2`, `Emote3`, `Emote4`, `EmoteDelay1`, `EmoteDelay2`, `EmoteDelay3`, `EmoteDelay4`, `RewardText`, `VerifiedBuild`) VALUES
+(1507, 0, 0, 0, 0, 0, 0, 0, 0, 'Hm... $N. You are still new to your path, but I sense the possibility for greatness in you.$B$BYou were born with gifts, $N. See that they do not go to waste.', 12340);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/create_sql.sh:19]: you added the SQL in the wrong file, it should be in the one below

**Review by LukasVolgger — APPROVED** [review]
Now it's fine. Can you please have a look at the locales for this too? Just check if the "Zarm" is there in the deDE or other locales and replace it with the variable.

**Comment by Melonio2** [comment]
> 
> 
> Now it's fine. Can you please have a look at the locales for this too? Just check if the "Zarm" is there in the deDE or other locales and replace it with the variable.

I searched for "Zarm" in the quest_offer_reward_locale table and I didn't find anything on my locale server, unlike when I searched in the quest_offer_reward where I can find it (I didn't applied my own "patch"). So I guess everything is good.

**Comment by LukasVolgger** [comment]
Ok, thx @Melonio2 !

**Comment by Kitzunu** [comment]
good work 💃. next time dont ignore the "PR template" :D

**Pull request merged**